Add decoding worker to reference and link (#4266)
EnterpriseDB/docs: product_docs/docs/pgd/5/reference/pgd-settings.mdx

Track lock timing when tracking statistics for relations.

## Decoding worker

### `bdr.enable_wal_decoder`

> Enables Logical Change Record (LCR) sending on a single node with a [decoding worker](../nodes#decoding-worker). By default, this is false. When set to true, a decoding worker process will be started and WAL senders will send the LCRs it produces. If set back to false, any WAL senders using LCR will be restarted and use the WAL directly.

Reviewer: ```suggestion
Enables Logical Change Record (LCR) sending on a single node with a [decoding worker](../nodes#decoding-worker). By default, this is false. When set to true, a decoding worker process starts and WAL senders send the LCRs it produces. If set back to false, any WAL senders using LCR are restarted and use the WAL directly.
```

Note: This setting also needs to be enabled on all nodes in the PGD group and for the `enable_wal_decoder` option to be set to true on the group.

### `bdr.receive_lcr`

> Enables the node to request Logical Change Records are used when subscribed to another node. By default, this is false. When this setting is true on a downstream node, the node will request that upstream nodes use Logical Change Records (LCR) when sending to it.  If `bdr.enable_wal_decoder` is set to true on a node, this setting should also be `true`.

Reviewer: ```suggestion
Enables the node to request Logical Change Records are used when subscribed to another node. By default, this is false. When this setting is true on a downstream node, the node requests that upstream nodes use Logical Change Records (LCR) when sending to it.  If `bdr.enable_wal_decoder` is set to true on a node, this setting should also be `true`.
```

Note: This setting also needs to be enabled on all nodes in the PGD group and for the `enable_wal_decoder` option to be set to true on the group.

### `bdr.lcr_cleanup_interval`

> Logical Change Record (LCR) file cleanup interval. When the [decoding worker](../nodes#decoding-worker) is enabled, the decoding worked stores LCR files as a buffer. These files are periodically cleaned and this setting controls the interval between any two consecutive cleanups. The default is 3 minutes. Setting it to zero will disable cleanup.

Reviewer: ```suggestion
Logical Change Record (LCR) file cleanup interval. When the [decoding worker](../nodes#decoding-worker) is enabled, the decoding worked stores LCR files as a buffer. These files are periodically cleaned and this setting controls the interval between any two consecutive cleanups. The default is 3 minutes. Setting it to zero disables cleanup.
```
